gridstatus/isone_api/isone_api.py

```diff
@@ -2029,12 +2029,22 @@ def _parse_fcm_reconfiguration_dataframe(
                 df[column] = pd.to_numeric(df[column], errors="coerce")
 
         if "ARA" in df.columns:
-            df["ARA"] = pd.to_numeric(df["ARA"], errors="coerce").astype("Int64")
+            if df["ARA"].notna().any() and (df["ARA"] != "").any():
```

Comment:
Does only the annual data have the ARA column? Would it be cleaner to add an `auction_type` parameter to the function and only add the ARA column if that value is "annual" instead of adding the column to the monthly data and then having to drop it later?